quiz/backend/players/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+from . import views
+
+app_name = "players"
+
+urlpatterns = [
+    path(
+        "players/username/", views.NicknameGeneratorAPIView.as_view(), name="nickname"
```

Comment:
How does it work that test for this url uses different url :smile:
Check `quiz/backend/players/tests/test_views.py:7`

Reply:
fixed 😞

Reply:
only think I've wondered about was the fact why the tests pass without changed path